src/pages/token/_symbol.vue

```diff
@@ -81,7 +81,7 @@ export default Vue.extend({
   },
   computed: {
     returnLink(): string | Route {
-      return this.fromRoute && this.fromRoute.fullPath !== this.$route.fullPath && this.fromRoute.path !== "/withdraw" ? this.fromRoute : "/account";
```

Comment:
this.fromRoute && this.fromRoute.fullPath can be simplified w/t the ts ? or !.
!this.fromRoute.path.includes("/transaction/") IMO should be replaced w/t .startsWith

BTW, since it's the third time same code appears, let's make a reusable utility closure/function